src/seedu/addressbook/storage/jaxb/AdaptedPerson.java

```diff
@@ -62,6 +64,10 @@ public AdaptedPerson(ReadOnlyPerson source) {
         address.isPrivate = source.getAddress().isPrivate();
         address.value = source.getAddress().value;
 
+        birthday = new AdaptedContactDetail();
+        birthday.isPrivate = source.getBirthday().isPrivate();
+        birthday.value = source.getBirthday().toString();
```

Comment:
Might want to make it consistent with the other columns like address/email/phone and just return a string value instead?